templates/tips/board.html

```diff
@@ -0,0 +1,67 @@
+{% extends "base.html" %} {%load static%} {%block content%}
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Tips</title>
+    <style type="text/css">
+      .author,
+      .eom {
+        color: grey;
+      }
+      .grid{
+          height:100%;
+          width:100%;
+      }
+      .item {
+        width: 50%;
+        background-color: grey;
+        position:relative;
+      }
+      .item p {
+        color: white;
+        text-align: center;
+      }
+      .item * {
+        color: white;
+        text-align: center;
+      }
+      .item--width2 {
+        width: 50%;
+      }
+      .date{
+          position: absolute;
+          top:0;
+          right:0;
+      }
+      .edit{
+        position: absolute;
+        bottom:0;
+        left:0;
+      }
+    </style>
+    <script src="https://unpkg.com/packery@2/dist/packery.pkgd.min.js"></script>
```

Comment:
what is that script doing?

Reply:
For the grid layout

Reply:
actually, i'm not sure if this is necessary either and I will remove it